src/Compilers/CSharp/Portable/Binder/Binder_Invocation.cs

```diff
@@ -1127,12 +1127,17 @@ private static bool IsUnboundGeneric(MethodSymbol method)
             return method.IsGenericMethod && method.ConstructedFrom() == method;
         }
 
+        private const int MaxCandidatesForErrorRecovery = 6;
```

Comment:
Can we get a comment on why the magic number 6 was chosen?  Even if it's arbitrary we should document that.